Sources/SwiftSyntax/SyntaxVisitor.swift.gyb

```diff
@@ -141,3 +141,75 @@ open class SyntaxVisitor {
     }
   }
 }
+
+public protocol SyntaxTransformVisitor {
```

Comment:
Let’s put this in a different file. `SyntaxRewriter` also has its dedicated file.